src/pandas_profiling/report/formatters.py

```diff
@@ -3,12 +3,23 @@
 import math
 import re
 from datetime import timedelta
-from typing import Any, Dict, List, Optional, Union
+from typing import Any, Callable, Dict, List, Optional, Union
 
 import numpy as np
 from markupsafe import escape
 
 
+def list_args(func: Callable) -> Callable:
```

Comment:
This could definitely use a docstring... By the looks of it, it... decorates the function `f(a, ...)` to return `map(f(...), a)` if argument `a` is a list..? Sounds weird.

Reply:
This was added for backwards compatibility. It simply extends the function to support providing the first argument as list. Ideally the data structure is consistent for a single and multiple reports, this is out of scope for this PR (it is sizable already).

Reply:
While there currently don't seem to be formatters that could "natively" work on lists, they'd need a special case with this decorator, too... 😕